openpype/hosts/blender/plugins/load/load_model.py

```diff
@@ -30,78 +21,81 @@ class BlendModelLoader(plugin.AssetLoader):
     label = "Link Model"
     icon = "code-fork"
     color = "orange"
+    color_tag = "COLOR_04"
 
-    def _remove(self, asset_group):
-        objects = list(asset_group.children)
+    _downstream_tasks = ("Rigging", "Modeling", "Texture", "Lookdev")
 
-        for obj in objects:
-            if obj.type == 'MESH':
-                for material_slot in list(obj.material_slots):
-                    bpy.data.materials.remove(material_slot.material)
-                bpy.data.meshes.remove(obj.data)
-            elif obj.type == 'EMPTY':
-                objects.extend(obj.children)
-                bpy.data.objects.remove(obj)
-
-    def _process(self, libpath, asset_group, group_name):
+    def _load_blend(self, libpath, asset_group):
```

Comment:
Can't all these `_load_blend` functions be DRYer? There is a lot of duplicate code accross all `load_.py` and `plugin.py` files.